helm/values-dev.yaml

```diff
@@ -79,16 +79,16 @@ microservice-chart:
     NODO_READ_TIMEOUT: "10000"
     NODO_CONNECTION_TIMEOUT: "10000"
     CARTS_MAX_ALLOWED_PAYMENT_NOTICES: "5"
-    ELASTIC_APM_SERVICE_NAME: pagopa-ecommerce-payment-requests-service
-    ELASTIC_APM_APPLICATION_PACKAGES: it.pagopa.ecommerce.payment.requests
-    ELASTIC_APM_SERVER_URL: http://quickstart-apm-http.elastic-system.svc:8200
-    ELASTIC_APM_LOG_SENDING: "false"
-    ECS_SERVICE_NAME: pagopa-ecommerce-payment-requests-service
-    ECS_SERVICE_ENVIRONMENT: "dev"
```

Comment:
restore this two env variables, they are used by logback encoder to encode those informations into json logs

Reply:
Ty :)

Reply:
Fixed in 8bf5fee